lib/osf-components/addon/components/paginated-list/has-many/component.ts

```diff
@@ -33,10 +33,6 @@ export default class PaginatedHasMany extends BaseDataComponent {
         const model = await taskFor(this.getModelTask).perform();
         if (this.usePlaceholders) {
             await taskFor(this.loadRelatedCountTask).perform(reloading);
-            // Don't bother querying if we already know there's nothing there.
-            if (this.totalCount === 0) {
-                return;
-            }
```

Comment:
Does the page still work with this back in? I think it would be nice to avoid unneeded queries if possible

Reply:
It doesn't work. The totalCount is 0.

Reply:
It's 0 with mirage or the real api? It may be we need to add `meta` into the mirage to get this to work there.

Reply:
I think the issue can/should be avoided by passing in `@usePlaceholders={{false}}` when we invoke this component. It seems that `usePlaceholders` should be false when we aren't using a `modelTaskInstance`